bayesian/utility/v2/ca_response_builder.py

```diff
@@ -18,11 +18,51 @@
 from urllib.parse import quote
 import logging
 from bayesian.utility.db_gateway import GraphAnalyses
-from bayesian.utils import version_info_tuple, convert_version_to_proper_semantic
+from bayesian.utils import version_info_tuple, convert_version_to_proper_semantic, \
+    server_create_analysis
+from typing import Dict, List, Tuple, Union, Set
+import re
+from collections import namedtuple
+from abc import ABC
+from flask import g
+
 
 logger = logging.getLogger(__name__)
 
 
+def validate_version(version: str) -> bool:
+    """Version should not contain special Characters."""
+    if re.findall('[!@#$%^&*()]', version):
+        return False
+    return True
+
+
+def unknown_package_flow(ecosystem: str, unknown_pkgs: Set[namedtuple], api_flow: bool) -> bool:
```

Comment:
Why is this defined in the response builder?

Reply:
Because it triggers as a Part of CA response.